crates/router/src/connector/square.rs

```diff
@@ -559,7 +559,7 @@ impl ConnectorIntegration<api::Capture, types::PaymentsCaptureData, types::Payme
         connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
         Ok(format!(
-            "{}v2/payments/{}/complete",
+            "{}v2/payments/{connector_payment_id}/complete",
```

Comment:
Hi @Ln11211 ,

This variable isn't declared anywhere , Please remove this changes